src/passengers_drop.c

```diff
@@ -0,0 +1,46 @@
+#include "passengers_drop.h"
+#include <assert.h>
+#include<stdio.h>
+#include<stdlib.h>
+#include <stdbool.h>
+#include "globals.h"
+/**
+ * The function takes in the elevator_array
+ *
+ * Removes the passenger from the floor and the elevator array.
+ *
+ * @return elevator array
+ **/
+int passengers_drop(struct elevator elevator_arr[], int index, struct passenger* floor_arr[], int cur_floor)
+{
+
+	assert(index <= elevator_count);
+	assert(cur_floor <= floor_count);
+	assert(cur_floor != elevator_arr[index].cur_floor);
+
+	
+	int passenger_dropped = elevator_arr[index].passenger_count;
```

Comment:
This should be max passengers. As we go over all the passengers

Reply:
Changes applied.